addons/html_editor/static/src/others/embedded_components/core/table_of_content/table_of_content.scss

```
@@ -8,13 +8,12 @@ $embedded-toc-bg--active: #1ba1e4;
     outline: 1px solid $o-gray-300;
     outline-offset: -1px;
     border-radius: 5px;
-    padding: 8px;
-    & > .o_embedded_toc_content {
+    background-color: $o-gray-100;
+    & > div > .o_embedded_toc_content {
```

Could simply be this then.
```suggestion
    & .o_embedded_toc_content {
```